source/shared/cpp/ObjectModel/ParseUtil.h

```diff
@@ -130,15 +130,16 @@ T ParseUtil::GetEnumValue(const Json::Value& json, AdaptiveCardSchemaKey key, T
 
         if (!propertyValue.isString())
         {
-            throw AdaptiveCardParseException("Enum type was invalid. Expected type string.");
+            throw AdaptiveCardParseException(ErrorStatusCode::InvalidPropertyValue, "Enum type was invalid. Expected type string.");
         }
 
         propertyValueStr = propertyValue.asString();
         return enumConverter(propertyValueStr);
     }
     catch (const std::out_of_range&)
     {
-        throw AdaptiveCardParseException("Enum type was out of range. Actual: " + propertyValueStr);
+        // throw AdaptiveCardParseException("Enum type was out of range. Actual: " + propertyValueStr);
```

Comment:
This will now add a warning instead of throwing an error.

Reply:
Add a comment here saying that just in case.

---
In reply to: [151812267](https://github.com/Microsoft/AdaptiveCards/pull/952#discussion_r151812267) [](ancestors = 151812267)

Reply:
Yeah, I just remembered I forgot to add the comment so I added it to the PR so you wouldn't be surprised by it 

---
In reply to: [151813511](https://github.com/Microsoft/AdaptiveCards/pull/952#discussion_r151813511) [](ancestors = 151813511,151812267)